src/getAppSyncConfig.js

```diff
@@ -229,3 +214,37 @@ export default function getAppSyncConfig(context, appSyncConfig) {
     mappingTemplates: makeMappingTemplates(cfg),
   };
 }
+
+async function invokeHandler(functionName, context, event) {
+  // Store deep copy of cliOptions so we can restore later
+  context.serverless.pluginManager.cliOptions["function"] = functionName;
+  context.serverless.pluginManager.cliOptions["data"] = JSON.stringify(event); // Capture stdout temporarily
+
+  let output = "";
+
+  // Save original write so we can restore later
+  process.stdout._orig_write = process.stdout.write;
+  process.stdout.write = (data) => {
+    output += data;
+    process.stdout._orig_write(data);
+  };
+
+  // Run serverless invoke local
+  await context.serverless.pluginManager.run(["invoke", "local"]);
```

Comment:
@theoribeiro  what does this do internally? 
I have a concern here. Let's say I am using [the webpack](https://github.com/serverless-heaven/serverless-webpack)
Currently, webpack live re-builds my lambdas as I save files. When Lambdas are invoked, the transpiled copy is run. (Thanks to the `location` config)

Invoking `sls invoke local` manually in the console, transpiles the function before running it. 

So, what would happened here? When will lambdas be built? Would it happen twice? (as I save, and when running?)